.github/workflows/cmake.yml

```diff
@@ -2,9 +2,13 @@ name: CMake
 
 on:
   push:
-    branches: [ main ]
+    branches:
+      - main
+      - temp_integ
```

Comment:
Do we still keep the temp_integ branch after this check in?

Reply:
We can remove it. We are planning to retire temp_integ anyway. Any PRs against it, need to be raised again main instead